src/pytorch_lightning/loggers/base.py

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Any
+
 import pytorch_lightning.loggers.logger as logger
 from pytorch_lightning.utilities.warnings import rank_zero_deprecation
 
 
-def rank_zero_experiment(*args, **kwargs) -> None:  # type: ignore[no-untyped-def]
+def rank_zero_experiment(*args, **kwargs) -> Any:  # type: ignore[no-untyped-def]
```

Comment:
You should be able to remove the `type: ignore`s. Also you will need to annotate the `args, kwargs` inputs

Reply:
I probably took your last comment too literally by using the future's args and kwargs.

Is something like what is shown below what you actually wanted me to do?

```python
def rank_zero_experiment(*args: Callable, **kwargs: Any) -> Any:
    rank_zero_deprecation(
        "The `pytorch_lightning.loggers.base.rank_zero_experiment` is deprecated in v1.7"
        " and will be removed in v1.9. Please use `pytorch_lightning.loggers.logger.rank_zero_experiment` instead."
    )
    return logger.rank_zero_experiment(*args, **kwargs)
```

```python
def merge_dicts(
    *args: Sequence[Mapping], **kwargs: Union[Optional[Mapping], Callable]
) -> Any:
    rank_zero_deprecation(
        "The `pytorch_lightning.loggers.base.merge_dicts` is deprecated in v1.7"
        " and will be removed in v1.9. Please use `pytorch_lightning.loggers.logger.merge_dicts` instead."
    )
    return logger.merge_dicts(*args, **kwargs)
```

Reply:
@carmocca It seems that both functions were left in place only to test the deprecation messages. I looked at the call references, and the only place rank_zero_experiment or merge_dicts is used, is in test_remove_1-9.py. Do *args and **kwargs need annotations if the user isn't interacting with the function, and if no logger is dependent on them?

Reply:
@awaelchli agreed. Is it fine to close this PR?